legal-api/migrations/versions/83a110e10979_add_batch_processing_id_to_colin_event_.py

```diff
@@ -0,0 +1,28 @@
+"""add_batch_processing_id_to_colin_event_ids
+
+Revision ID: 83a110e10979
+Revises: d55dfc5c1358
+Create Date: 2024-10-07 15:34:55.319366
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '83a110e10979'
+down_revision = 'd55dfc5c1358'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('colin_event_ids', sa.Column('batch_processing_id', sa.Integer(), nullable=True))
+    op.add_column('colin_event_ids', sa.Column('batch_processing_step', sa.Enum('WARNING_LEVEL_1', 'WARNING_LEVEL_2', 'DISSOLUTION', name='batch_processing_step'), nullable=True))
```

Comment:
Something I found interesting and didn't know about how alembic and sqlalchemy handles re-using enums:

To me it looked like you were re-creating the enum, so I was wondering what would happen if I changed the enum values here, and if it would have any effect on the existing `batch_processing_step` enum.

<img width="1002" alt="Screenshot 2024-10-09 at 10 15 58 AM" src="https://github.com/user-attachments/assets/4330aa9e-e52c-4bba-afc1-c93a31309da8">

Turns out this has no effect on the values `batch_processing.batch_processing_step` or `colin_event_id.batch_processing_step`. So alembic must be pointing the `colin_event_id.batch_processing_step` enum to the existing one.

Turns out, you can remove all the enum values here and it still works as long as the `name` parameter points to the correct enum.

<img width="961" alt="Screenshot 2024-10-09 at 10 19 21 AM" src="https://github.com/user-attachments/assets/f235cc37-f462-4697-9398-8654dfaa2657">

Can you update this line to remove the enum values? Just to avoid any confusion as to what this line is doing.

Reply:
Good Catch.
I removed it and tested it. It works well